content/api/commands/origin.md

```diff
@@ -383,81 +354,6 @@ and reuse it across tests.
 
 ## Notes
 
-### Migrating existing tests
```

Comment:
Do we want to keep around the migration documentation? That seems like it might be still useful

Reply:
Yeah I think that makes sense. I just wanted to make sure this doesn't get lost.

Reply:
Moved the content to a migration guide. [dde69c1](https://github.com/cypress-io/cypress-documentation/pull/4807/commits/dde69c1ebf2d547bc1f334b38734966b63196da2)